jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -428,23 +596,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 26,
    "metadata": {
     "scrolled": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"_links\": {\"_self\": \"https://api.planet.com/compute/ops/orders/v2/eb65f433-61fd-4835-a2ad-2d6d007494f0\"}, \"created_on\": \"2023-06-22T11:01:14.824Z\", \"error_hints\": [], \"id\": \"eb65f433-61fd-4835-a2ad-2d6d007494f0\", \"last_message\": \"Preparing order\", \"last_modified\": \"2023-06-22T11:01:14.824Z\", \"name\": \"test_order_sdk_method_2\", \"products\": [{\"item_ids\": [\"20200111_145817_0f1a\", \"20200111_145816_1_0f1a\"], \"item_type\": \"PSScene\", \"product_bundle\": \"analytic_sr_udm2\"}], \"state\": \"queued\", \"tools\": [{\"clip\": {\"aoi\": {\"coordinates\": [[[-93.299129, 42.699599], [-93.299674, 42.812757], [-93.288436, 42.861921], [-93.265332, 42.924817], [-92.993873, 42.925124], [-92.993888, 42.773637], [-92.998396, 42.754529], [-93.019154, 42.699988], [-93.299129, 42.699599]]], \"type\": \"Polygon\"}}}, {\"bandmath\": {\"b1\": \"(b4 - b3) / (b4+b3)\", \"pixel_type\": \"32R\"}}]}\n"
+     ]
+    }
+   ],
    "source": [
     "!planet orders create 'json_request.txt'"
```

Comment:
It's pretty strange we have a notebook which mixes the python SDK with the CLI. @mkshah605 what's up? Are we hoping to keep this? Or was this just a temporary solution?

Reply:
creating a new PR for this issue